### PR TITLE
Set the default Java version of grind to 7 for backward compatibility.

### PR DESCRIPTION
**Pull request by flyrain**
Change default Java version to 7 for backward compatibility. Check CDH-48954 for the details.

### CODE REVIEW

File: docs/grind.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Per-project Configuration
 * `file_globs`: Specifies additional test dependencies via a comma-delimited list of Unix-style path globs. These globs are interpreted by Python's [glob.iglob](https://docs.python.org/2/library/glob.html#glob.iglob).
 * `file_patterns`: Specifies additional test dependencies via a comma-delimited list of filename patterns. These patterns are interpreted by Python's [fnmatch.fnmatch](https://docs.python.org/2/library/fnmatch.html#fnmatch.fnmatch).
 * `artifact_archive_globs`: Specifies test output to upload after a test has run, via comma-delimited Python glob.iglob glob strings. By default, this matches Surefire's test XML output (`**/surefire-reports/TEST-*.xml`), but it can be modified to also upload additional logs.
-* `java_version`: Chooses the runtime JDK version. Supported values are 7 or 8.
+* `java_version`: Chooses the runtime JDK version. Supported values are 7 or 8, the default is 7.
```
Comment (umbrant) [inline docs/grind.md]: the example output below also needs to be updated to say 7 instead of 8

Comment (flyrain) [inline docs/grind.md]: Thanks for the review. Changed the example output to 7. What test do you mean by saying "comment on testing"?

Comment (umbrant) [inline docs/grind.md]: Just asking if you've run grind manually and confirmed that the behavior is as expected.

Comment (flyrain) [inline docs/grind.md]: Manual test is done. Looks good.

**Comment by umbrant** [comment]
Thanks for the contribution!

**Comment by flyrain** [comment]
Thanks for the review and advice.

**Pull request merged**